Tests/Validator/UniqueUrlValidatorTest.php

```diff
@@ -99,6 +102,7 @@ public function testValidateWithRootUrlAndNoParent()
 
     public function testValidateWithPageDynamic()
     {
+        $this->skipInPHP55();
```

Comment:
No, I'm afraid I don't understand what is happening here. See https://travis-ci.org/sonata-project/SonataPageBundle/jobs/228055349#L463